sys/net/network_layer/ng_ipv6/netif/ng_ipv6_netif.c

```diff
@@ -147,13 +150,14 @@ int ng_ipv6_netif_add_addr(kernel_pid_t pid, const ng_ipv6_addr_t *addr,
         return -ENOENT;
     }
 
-    if ((addr == NULL) || (ng_ipv6_addr_is_unspecified(addr))) {
+    if ((addr == NULL) || (ng_ipv6_addr_is_unspecified(addr)) ||
+        (prefix_len < 1) || (prefix_len > 128)) {
```

Comment:
_always_ choose readability before optimizations. The compiler will probably choose the fastest evaluation of the condition anyway.